src/shipit_code_coverage/shipit_code_coverage/taskcluster.py

```diff
@@ -20,14 +21,16 @@ def _get_build_platform_name(platform):
 
 
 def get_last_task(platform):
-    r = requests.get(index_base + 'task/gecko.v2.mozilla-central.latest.firefox.{}'.format(_get_build_platform_name(platform)))
+    index = get_service('index')
+    r = index.findTask(index_base + 'task/gecko.v2.mozilla-central.latest.firefox.{}'.format(_get_build_platform_name(platform)))
     r.raise_for_status()
     last_task = r.json()
```

Comment:
I think both `raise_for_status` and `json` are handled by taskcluster-client directly, so you shouldn't need them (infact, they shold fail here as `r` is not a Response object).

Reply:
thanks, looks like it will respond with a object of type 'http://schemas.taskcluster.net/index/v1/indexed-task-response.json#' 
Will make changes accordingly